src/components/Divider/src/index.tsx

```diff
@@ -35,7 +17,16 @@ export interface DividerProps extends BaseDataDisplayClassesProps {
  * @constructor Constructs an instance of Divider.
  */
 export const Divider: React.FC<DividerProps> = (props: DividerProps) => {
-  return <MaterialDivider {...props} />;
+  const classes = {
+    root: 'denhaag-divider',
+    vertical: 'denhaag-divider--vertical'
+  }
+
+  return (
+    <StylesProvider injectFirst>
+      <MaterialDivider variant={"fullWidth"} classes={classes} role={"presentation"} {...props} />
```

Comment:
We're not using `...props` anymore. It risks exposing MaterialUI API we don't want to support. Pass every prop in our interface explicitly.